spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -90,6 +90,29 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     }
   }
 
+  test("bitwise_count") {
+    Seq(false, true).foreach { dictionary =>
+      withSQLConf("parquet.enable.dictionary" -> dictionary.toString) {
+        val table = "bitwise_count_test"
+        withTable(table) {
+          sql(s"create table $table(col1 long, col2 int, col3 short, col4 byte) using parquet")
+          sql(s"insert into $table values(1111, 2222, 17, 7)")
```

Comment:
Do you mind adding random number cases?

Reply:
Yep. Added tests with random data.

Reply:
Can we also add a test with a Parquet file not created by Spark (see `makeParquetFileAllTypes`) and see if this reports correct results with unsigned int columns?

Reply:
`CometTestBase.makeParquetFileAllTypes` has all the integer types you may want to test.